platform_versioned_docs/version-24.1.0/enterprise/configuration/configtables/features_env.yml

```diff
@@ -19,3 +19,8 @@
   Description: > 
     Use AWS Simple Email Service (SES) to send Seqera emails instead of SMTP.
   Value:                'Default: `false`'
+- 
+  Environment variable:            '`TOWER_ALLOW_NEXTFLOW_LOGS`'
+  Description: > 
+    Allow log and report files from Nextflow CLI runs (`-with-tower`) to be included in Seqera reports. Runs must be launched from a workspace primary compute environment. 
```

Comment:
```suggestion
    Allow log and report files from Nextflow CLI runs (`-with-tower`) to be included in Seqera reports. Runs must be launched from a workspace's primary compute environment. 
```

Reply:
the logs will not be included in the reports. Simply, they will be made available in the UI vs not being accessible.  
The runs are launched locally on HPC or AWS without using any Tower CE
What happens is that the files (logs and reports) must be accessible to the primary CE in the workspace so tower will be able to fetch them successfully.